app/qml/editor/inputtextedit.qml

```diff
@@ -77,6 +77,15 @@ Item {
     onEditingFinished: {
       valueChanged( text, text === undefined )
     }
+
+    //! Commit value if has changed when widget gets out of the FeatureForm (ListView) viewport
+    Component.onDestruction: {
+      if ( textField.activeFocus ) {
+        if ( value !== textField.text ) {
+          valueChanged( textField.text, textField.text === undefined )
```

Comment:
Shouldn't we rather use `displayText` property? I think `text` does not need to be committed

```suggestion
        if ( value !== textField.displayText ) {
          valueChanged( textField.displayText, textField.displayText === undefined )
```

Reply:
It is save to change it to displayText. However there was no such problem with text or using onTextChanged, the issue was related to not committing the change to a feature.

Have you experienced any issue during the testing?